ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -291,6 +298,21 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
     previousCall = { content: 'Identical Call', hover: 'Identical call in previous callset', color: 'blue' }
   } else if (genotype.prevOverlap) {
     previousCall = { content: 'Overlapping Call', hover: 'Overlapping call in previous callset', color: 'teal' }
+  } else if (Number.isInteger(genotype.prevNumAlt)) {
+    const hasSameHemiXGenotype = isHemiX && (genotype.numAlt === 1 || genotype.numAlt === 2) &&
+      (genotype.prevNumAlt === 1 || genotype.prevNumAlt === 2)
```

Comment:
I need clarification about the logic for the same hemiX genotype. Does it mean it is a hemiX with the same `numAlt` and `prevNumAlt` values?

Reply:
For hemiX num alt of 1 and 2 are displayed the exact same so to the user they would be unable to see that anything had changed, so this is checking for that case